### ISSUE TITLE
How did you train resnet50 model

### ISSUE THREAD

**Issue by mingxiaoh** [Cadene/pretrained-models.pytorch#148]
hello, I am trying to train resnet50 based on pytorch, however, I only got 57% top1 validation acuracy on imagenet, which is far from SOTA. I tried to do inference using the pretrained resnet50 model you provided, it got 73% validation accuracy on imagenet. So, I would like to know, how did you train resnet50? Would you please share the training script(or hyper parameters setting) with me? Thanks in advance.

**Comment by ZarkDing** [comment]
Hi,Could you please send your code to me for learning？My Email is sa517059@mail.ustc.edu.cn.Thank you very much!
